src/appendFrames.ts

```diff
@@ -58,6 +58,8 @@ export function appendMatchingFrames(prev: DataFrame[], b: DataFrame[]): DataFra
       for (let i = 0; i < f.length; i++) {
         for (let idx = 0; idx < old.fields.length; idx++) {
           old.fields[idx].values.add(f.fields[idx].values.get(i));
+          // FIXME: I assume this is meant to modify `out` but it's
+          // not doing anything atm
```

Comment:
lol should we make a ticket or something?

Reply:
double checking this it actually works :exploding_head: 

I assume that `old.fields[idx]` is pointing to the same vector reference than `out` so modifying that modifies the result. I have added a test that covers this.